media-gfx/xsane/xsane-0.999-r4.ebuild

```diff
@@ -77,14 +75,14 @@ src_install() {
 	# link xsane so it is seen as a plugin in gimp
 	if use gimp; then
 		local plugindir
-		local gimptool=( "${EPREFIX}"/usr/bin/gimptool* )
+		local gimptool=( "${ESYSROOT}"/usr/bin/gimptool* )
```

Comment:
I think EPREFIX might be right for plugins?

Reply:
yes, and if it's from a prefix installation, it will still work?